### PR TITLE
Add jquery-appear@0.1 w/ git auto-update

### LINKED ISSUE #9525: [Request] jquery.appear
Library name: jquery.appear
Git repository url: https://github.com/bas2k/jquery.appear/
License(s): MIT

### PR DESCRIPTION
**Pull request by sashberd**
Pull request for issue: #9525
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bas2k/jquery.appear
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 19
   - Count of stars: 213
   - Count of forks: 75
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery-appear/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-appear",
+  "filename": "jquery.appear.min.js",
+  "version": "0.1",
+  "description": "jQuery plugin to call a function when an element appears",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bas2k/jquery.appear.git"
+  },
+  "author": "Alexander Brovikov",
```
Comment (extend1994) [inline ajax/libs/jquery-appear/package.json]: Maybe one more Michael Hixson from https://github.com/bas2k/jquery.appear/blob/master/jquery.appear.js#L7?

File: ajax/libs/jquery-appear/0.1/jquery.appear.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+/*
```
Comment (sufuf3) [inline ajax/libs/jquery-appear/0.1/jquery.appear.js]: This file is different from upstream.
This file's md5sum is `cbbd68c06c9cdb2481b7050f6e64375e`.
Upstream's is `5a457d262e3c32d25c003ca412ee7fe6`
Please re-add the file.
Thanks.

File: ajax/libs/jquery-appear/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-appear",
+  "filename": "jquery.appear.min.js",
+  "version": "0.1",
+  "description": "jQuery plugin to call a function when an element appears",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bas2k/jquery.appear.git"
+  },
+  "author": "Michael Hixson",
```
Comment (sufuf3) [inline ajax/libs/jquery-appear/package.json]: Maybe the author should be `bas2k`.
What do you think?
Thanks

**Comment by sashberd** [comment]
@sufuf3 The PR was updated.

There are 3 authors of this library. @extend1994 Asked me to change to different one, then you asked to change it. So I added all 3 of them... What do you think about it?
Also I used wget in order to get file from repo.
FYR

**Comment by sashberd** [comment]
@sufuf3 I re add file with wget, I hope no this symbol is eliminated cause I cannot see it in my tools

**Comment by sashberd** [comment]
@sufuf3 ping

**Pull request merged**